js/main.js

```diff
@@ -58,34 +57,35 @@ function generateFour() {
   console.log('List of four Pokemon:', data.currentFour)
   data.currentPokemon = allPokemonList[data.currentFour[0] - 1] //Subtract 1
 }
-generateFour() //Array of 4 numbers, we want to reset this later.
+generateFourRandomPokemonNumbers() //Array of 4 numbers, we want to reset this later.
 
-var tenSecondsBar = null;
+var xhr = null;
```

Comment:
You shouldn't make `xhr` global like this. Is this so that you can access `xhr.response.sprites.other['official-artwork'].front_default` in `appendPokemonPicture`? If so, that means you should pass that value as an argument to the function where you call it.